script/repl

```diff
@@ -4,12 +4,7 @@
 
 set -e
 
-cd $(dirname "$0")/..
+cd "$(dirname "$0")/.."
 
-repl_builddir=dist-repl
-
-if [[ ! -d $repl_builddir ]]; then
-  echo "$repl_builddir does not exist, first run 'cabal repl --builddir=$repl_builddir', exit, and then re-run $0"
```

Comment:
This is no longer necessary, but it _might_ be necessary to do a `cabal build` (of at least part of the project) to populate the package db. However, in those cases the error messages you see inform you of that fact already so there isn’t much benefit to adding an extra check.